test/test_oneview_logical_switch.py

```diff
@@ -35,161 +34,149 @@
                                    {'logicalSwitchManagementHost': '172.16.1.2'}]
 )
 
+PARAMS_FOR_PRESENT = dict(
```

Comment:
Just asking. It's not a big deal, since it's visible only inside the module, but for me it looks weird since we are using the Python OO style,